arrays.dd

```diff
@@ -346,16 +346,17 @@ a[] = b[] + 4;
 ---
 
         $(P A vector operation is indicated by the slice operator appearing
-        as the lvalue of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |= operator.
-        The rvalue can be an expression consisting either of an array
-        slice of the same length and type as the lvalue or an expression
-        of the element type of the lvalue, in any combination.
+        as the left-hand side of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |=
+        operator.
+        The right-hand side can be an expression consisting either of an array
+        slice of the same length and type as the left-hand side or an expression
+        of the element type of the left-hand side, in any combination.
         The operators supported for vector operations are the binary
         operators +, -, *, /, %, ^, $(AMP) and |, and the unary operators
         - and ~.
         )
 
-        $(P The lvalue slice and any rvalue slices must not overlap.
+        $(P The slice on the left and any slices on the right must not overlap.
```

Comment:
"any" makes sense to me. I take it to mean that if there are *any* slices on the right, then they must not overlap with the one on the left.

"all" sounds weirder to me. Would that mean that the the slices on the right must not overlap with each other? Is that so? dmd doesn't mind it.

I don't know if right-hand slices may overlap with each other, and I'm not a native speaker, so I'd like to leave this for someone else, if it needs changing.